src/mon/Monitor.cc

```diff
@@ -536,14 +536,14 @@ int Monitor::preinit()
   assert(!logger);
   {
     PerfCountersBuilder pcb(g_ceph_context, "mon", l_mon_first, l_mon_last);
-    pcb.add_u64(l_mon_num_sessions, "num_sessions");
-    pcb.add_u64_counter(l_mon_session_add, "session_add");
-    pcb.add_u64_counter(l_mon_session_rm, "session_rm");
-    pcb.add_u64_counter(l_mon_session_trim, "session_trim");
-    pcb.add_u64_counter(l_mon_num_elections, "num_elections");
-    pcb.add_u64_counter(l_mon_election_call, "election_call");
-    pcb.add_u64_counter(l_mon_election_win, "election_win");
-    pcb.add_u64_counter(l_mon_election_lose, "election_lose");
+    pcb.add_u64(l_mon_num_sessions, "num_sessions", "Current number of opened monitor sessions");
+    pcb.add_u64_counter(l_mon_session_add, "session_add", "Number of created monitor sessions");
+    pcb.add_u64_counter(l_mon_session_rm, "session_rm", "Number of remove_session calls in monitor");
+    pcb.add_u64_counter(l_mon_session_trim, "session_trim", "Number of trimed monitor sessions");
+    pcb.add_u64_counter(l_mon_num_elections, "num_elections", "Number of elections monitor took part in");
+    pcb.add_u64_counter(l_mon_election_call, "election_call", "Number of elections started by monitor");
+    pcb.add_u64_counter(l_mon_election_win, "election_win", "Number of elections won by monitor");
+    pcb.add_u64_counter(l_mon_election_lose, "election_lose", "Number of elections lost by monitor");
```

Comment:
perf counters are namespaced into a service, so the "by monitor" parts here are redundant